packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts

```diff
@@ -219,7 +219,7 @@ export class AuditLogManager {
               <mj-body>
                 <mj-section>
                   <mj-column>
-                    <mj-image width="150px" src="https://graphql-hive.com/logo.png"></mj-image>
+                    <mj-image width="200px" src="https://the-guild.dev/graphql/hive/github-org-image.png"></mj-image>
```

Comment:
```suggestion
                    <mj-text
           font-size="16"
           font-weight="500">Hive</mj-text>
```

Reply:
![Screenshot 2025-02-20 at 16 25 26](https://github.com/user-attachments/assets/8a1f39b8-174d-468d-bca3-aee3c48e7f1b)
![Screenshot 2025-02-20 at 16 24 58](https://github.com/user-attachments/assets/7add8c3e-8c99-4133-a965-60d4bfa85109)

I think instead of the green banner, we could just render "Hive" and leave it for future improvement.
I have one concern about the image. It's hosted on a different domain than the email address. I think it could affect the spam filter.